docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-aws.adoc

```diff
@@ -19,44 +19,60 @@ Complete the steps below to configure agentless scanning based on the needs of y
 
```

Comment:
we talked about fixing this, but i don't see it in the code:
![image](https://github.com/hlxsites/prisma-cloud-docs/assets/24986085/f8ac7032-42f8-45d9-8937-4d3db961ad8e)